plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -29,11 +35,11 @@ open class KtlintExtension {
     var ignoreFailures = false
 
     /**
-     * Report output format.
+     * Report output formats.
      *
      * Available values: plain, plain_group_by_file, checkstyle, json.
      *
-     * Default is plain.
+     * Default is empty.
      */
-    var reporter: ReporterType = ReporterType.PLAIN
+    var reporters: List<ReporterType> = mutableListOf()
```

Comment:
Very much a breaking change. But it makes sense.
Perhaps this should be a `set` instead of a `list`? Unless ordering really matters?

Reply:
This collection should keep `ReporterType.PLAIN` by default.